source/_integrations/switchbot_cloud.markdown

```
@@ -182,6 +187,15 @@ Features:
 - get temperature
 - get humidity
 
+### Fans
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)